#! /bin/sh /usr/share/dpatch/dpatch-run
## pod-perl-5.18.dpatch by gregor herrmann <gregoa@debian.org>
##
# Description: add =encoding to POD. perl 5.18 is more picky and exists with an error.
# Origin: vendor
# Bug-Debian: http://bugs.debian.org/708718
# Forwarded: no
# Author: gregor herrmann <gregoa@debian.org>
# Last-Update: 2013-05-19

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' collectd-5.1.0~/src/collectd-java.pod collectd-5.1.0/src/collectd-java.pod
--- collectd-5.1.0~/src/collectd-java.pod	2012-04-02 10:04:58.000000000 +0200
+++ collectd-5.1.0/src/collectd-java.pod	2013-05-19 16:18:19.323246574 +0200
@@ -1,3 +1,5 @@
+=encoding utf8
+
 =head1 NAME
 
 collectd-java - Documentation of collectd's "java plugin"
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' collectd-5.1.0~/src/collectd.conf.pod collectd-5.1.0/src/collectd.conf.pod
--- collectd-5.1.0~/src/collectd.conf.pod	2012-04-02 10:04:58.000000000 +0200
+++ collectd-5.1.0/src/collectd.conf.pod	2013-05-19 16:17:52.395246170 +0200
@@ -1,3 +1,5 @@
+=encoding utf8
+
 =head1 NAME
 
 collectd.conf - Configuration for the system statistics collection daemon B<collectd>
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' collectd-5.1.0~/src/collectdctl.pod collectd-5.1.0/src/collectdctl.pod
--- collectd-5.1.0~/src/collectdctl.pod	2012-04-02 10:04:58.000000000 +0200
+++ collectd-5.1.0/src/collectdctl.pod	2013-05-19 16:18:04.251246348 +0200
@@ -1,3 +1,5 @@
+=encoding latin1
+
 =head1 NAME
 
 collectdctl - Control interface for collectd
